components/canvas/webgl_thread.rs

```diff
@@ -64,6 +65,9 @@ pub struct WebGLThread<VR: WebVRRenderHandler + 'static> {
     webvr_compositor: Option<VR>,
     /// Texture ids and sizes used in DOM to texture outputs.
     dom_outputs: FnvHashMap<webrender_api::PipelineId, DOMToTextureData>,
+    /// Tracking GL contexts that must not be modified but can still be
+    /// targeted by WebGL commands.
```

Comment:
This should document that this map only has keys for contexts that are currently locked, _and_ for which a `PreventModification` message has been sent.